### PR TITLE
Updated readme to how to integrate using artifactory

### PR DESCRIPTION
**Pull request by yigitozenaccedo**
Added Artifactory upload task

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,20 +42,51 @@ You may also want to refer to the [Accedo One Rest API documentation](https://de
...
-	maven {
-		url  "https://dl.bintray.com/tibor-pasztor-accedo/accedo-products"
-	}
+
+</p>
+</details> 
+
+
+<details>
+<summary>Setup using Bintray (Which will be deprecated soon!)</summary>
```
Comment (petersanta) [inline README.md:89]: The Control SDK is public on github and can be used by 3rd parties. What are the plans, how they can access to it in the future?

Comment (yigitozenaccedo) [reply README.md:89]: I will create a ticket for IT to check if we can allow anonymous access to repo (https://jfrog.com/knowledge-base/how-can-i-allow-anonymous-user-access-to-specific-repositories-only/)

Comment (petersanta) [reply README.md:89]: Cool, thank you!

Comment (yigitozenaccedo) [reply README.md:89]: As they say, we won't serve artifacts anonymously :/ https://accedobroadband.jira.com/browse/IT-20515

Right now only way would be to open a repository in Sonatype but Idk if they are going to allow it 🤷 

If not, we serve aar

**Pull request merged**